runners/tez/src/test/java/org/apache/beam/runners/tez/TezRunnerTest.java

```diff
@@ -0,0 +1,138 @@
+package org.apache.beam.runners.tez;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.io.TextIO;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.GroupByKey;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.KV;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests for the Tez runner.
+ */
+public class TezRunnerTest {
+
+  private static final String TOKENIZER_PATTERN = "[^\\p{L}]+";
+  private static final String INPUT_LOCATION = "src/test/resources/test_input.txt";
```

Comment:
<!--new_thread; commit:346274d62540a16c9bf795fc6e9ab2cd28430008; resolved:0-->
TBH it would probably be simpler to use a @Rule TemporaryDirectory and dump the contents there, since it is just a tiny amount of data.